packages/dds/merge-tree/src/mergeTree.ts

```diff
@@ -1943,6 +1958,117 @@ export class MergeTree {
         }
     }
 
+    /**
+     * Revert an unacked local op
+     */
+    public rollback(op: any, localOpMetadata: unknown) {
```

Comment:
nit. i generally prefer merge tree to use strong types if we have them, and force the caller, client to do any type coercion necessary